modules/nixos/monitoring/grafana.nix

```diff
@@ -0,0 +1,38 @@
+{ config, ... }:
+{
+  services.grafana = {
+    enable = true;
+    settings = {
+      analytics.reporting_enabled = false;
+
+      "auth.anonymous".enabled = true;
```

Comment:
We could use the github org for authentication here. Than we don't need to create accounts manually.